### PR TITLE
Feature/moz 374 additional events fields

### PR DESCRIPTION
**Pull request by kycapstick**
Adding dropdown to Event Creation flow for preferred language.

### CODE REVIEW

**Comment by kycapstick** [comment]
Designs were finalized for event goals and projected attendees so added those to this PR

File: plugins/events-manager/forms/event-editor.php [hunk 1]
```diff
@@ -51,8 +57,21 @@
...
-                <input class="event-creator__input event-creator__input" type="text" name="event_name" id="event-name" required value="<?php echo esc_attr($EM_Event->event_name,ENT_QUOTES); ?>" />
+				<div class="event-creator__three-up">
+					<div class="wide--double">
+						<label class="event-form-name event-creator__label" for="event-name"><?php print __( 'Event Name *', 'commuity-portal'); ?></label>
+						<input class="event-creator__input event-creator__input" type="text" name="event_name" id="event-name" required value="<?php echo esc_attr($EM_Event->event_name,ENT_QUOTES); ?>" />
+					</div>
+					<div class="wide wide--md-third">
+						<label class="event-creator__label" for="language"><?php print __('Language') ?></label>
+						<select class="event-creator__dropdown" name="language" id="language">
+							<option value disabled selected></option>
```
Comment (dnaluz) [inline plugins/events-manager/forms/event-editor.php]: No default value

**Comment by dnaluz** [comment]
Screen Shot 2020-03-16 at 11 26 52 AM

Broken when there is an error

**Comment by dnaluz** [comment]
@kycapstick looks like there is still some broken styles here.

Screen Shot 2020-03-17 at 9 59 57 AM

**Pull request merged**